worldengine/simulations/precipitation.py

```diff
@@ -42,18 +47,61 @@ def _calculate(seed, width, height):
 
         for y in range(height):#TODO: numpy
             y_scaled = float(y) / height
+            n_scale = 1024 / float(height)
             latitude_factor = 1.0 - (abs(y_scaled - 0.5) * 2)
             for x in range(width):
-                n = snoise2(x / freq, y / freq, octaves, base=base)
+                n = snoise2((x * n_scale) / freq, (y * n_scale) / freq, octaves, base=base)
 
                 # Added to allow noise pattern to wrap around right and left.
                 if x < border:
-                    n = (snoise2(x / freq, y / freq, octaves,
+                    n = (snoise2( (x * n_scale) / freq, (y * n_scale) / freq, octaves,
                                  base=base) * x / border) + (
-                        snoise2((x + width) / freq, y / freq, octaves,
+                        snoise2(( (x * n_scale) + width) / freq, (y * n_scale) / freq, octaves,
                                 base=base) * (border - x) / border)
 
-                precipitation = (latitude_factor + n * 4) / 5.0
-                precipitations[y, x] = precipitation
+                precipitations[y, x] = n
```

Comment:
Is ```latitude_factor``` now obsolete? If so, please remove it.
I am interested to see the results of these changes. :)